R/methods-epi_archive.R

```diff
@@ -165,6 +181,17 @@ epix_merge = function(x, y, ..., locf = TRUE, nan = NA) {
 #' 
 #' @importFrom rlang enquo
 #' @export
+#' @examples 
+#' # every date is a reference time point for the 3 day average sliding window
```

Comment:
@ChloeYou

Reply:
Hi Logan! Can you elaborate on the data latency part? Is it because the data is always one day late, hence even when n = 3, the average is taken from the past 2 days. Would this be the same case for n = 7, where for most dates if is calculating the average of the past 6 days? Thanks! @brookslogan

Reply:
I'm not sure if it's "always" or "almost always", but yes, data is always one day late, in some sense... The first time that data "about" day (`time_value`) t is first made available is after day t ends; for some data sources, this is typically sometime on day (`as_of`) t+1, and for others, it's later.  It depends on the definition of "about", the upstream data provider's/providers' data pipelines, weekends, COVIDcast's data pipeline, and maybe some other details.  Yes, if `n=7`, I would expect this typically to be calculating an average of 6 values for JHU-CSSE (but it looks like for doctor-visits it will be less).

(And "about" can get a bit complicated.  E.g., for JHU-CSSE data: since it's data republished by COVIDcast, republished from JHU-CSSE from state&local reports and correspondence, and in most instances involves taking differences in cumulative cases&deaths *reported by state&local through time t* rather than *happening on day t*, but with special exceptions in some locations, times, and versions.  The doctor-visit data is a little bit cleaner regarding "about" because it starts with raw data about doctor visits that actually occurred on t, although then this is smoothed.)

Reply:
Thank you for clarifying! Really appreciate it :)